### ISSUE TITLE
SG2086 - useless to double quote exit

### ISSUE THREAD

**Issue by MatteoCollica** [koalaman/shellcheck#1961]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2086
- My shellcheck version (`shellcheck --version` or "online"): 0.7.1
- [V] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [V] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [V] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [V] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
exit $1

```

#### Here's what shellcheck currently says:
Double quote to prevent globbing and word splitting.

#### Here's what I wanted or expected to see:
Nothing, that's useless to double quote a single number

**Comment by koalaman** [comment]
ShellCheck is correct in this case. Since this value comes from whatever invokes the script, there is no guarantee that it's a single number.
